chapters/publication.tex

```diff
@@ -407,73 +391,82 @@ \subsection{Publishing data for replication}
 The \texttt{sdcMicro} tool, in particular, has a feature
 that allows you to assess the uniqueness of your data observations,
 and simple measures of the identifiability of records from that.
-Additional options to protect privacy in data that will become public exist,
-and you should expect and intend to release your datasets at some point.
-One option is to add noise to data, as the US Census Bureau has proposed,\cite{abowd2018us}
-as it makes the trade-off between data accuracy and privacy explicit.
-But there are no established norms for such ``differential privacy'' approaches:
+
+There will almost always be a trade-off between accuracy and privacy.
+For publicly disclosed data, you should favor privacy.
+Stripping identifying variables from a dataset may not be sufficient to protect respondent privacy,
+due to the risk of re-identification. 
+One potential solution is to add noise to data, as the US Census Bureau has proposed.\cite{abowd2018us}
+This makes the trade-off between data accuracy and privacy explicit.
+But there are not, as of yet, established norms for such ``differential privacy'' approaches:
 most approaches fundamentally rely on judging ``how harmful'' information disclosure would be.
 The fact remains that there is always a balance between information release (and therefore transparency)
 and privacy protection, and that you should engage with it actively and explicitly.
 The best thing you can do is make a complete record of the steps that have been taken
 so that the process can be reviewed, revised, and updated as necessary.
 
+In cases where PII data is required for analysis,
+we recommend embargoing the sensitive variables when publishing the data.
+Access to the embargoed data could be granted for specific purposes,
+such as a computational reproducibility check required for publication,
+if done under careful data security protocols and approved by an IRB.
+
+
 \subsection{Publishing code for replication}
 
 Before publishing your code, you should edit it for content and clarity
 just as if it were written material.
 The purpose of releasing code is to allow others to understand
 exactly what you have done in order to obtain your results,
 as well as to apply similar methods in future projects.
-Therefore it should both be functional and readable.
-If you've followed the recommendations in this book,
-this will be much easier to do.
+Therefore it should both be functional and readable
+(if you've followed the recommendations in Chapter Six,
```

Comment:
I think "this book" is better as it also includes folder structures in data management etc.

Reply:
```suggestion
(if you've followed the recommendations in this book
```